tests/integration/test_phir_64_bit.py

```diff
@@ -0,0 +1,35 @@
+from pecos.engines.hybrid_engine import HybridEngine
+
+
+def bin2int(result: list[str]) -> int:
+    return int(result[0], base=2)
+
+
+def test_setting_cvar():
+    phir = {
+        "format": "PHIR/JSON",
+        "version": "0.1.0",
+        "ops": [
+            {"data": "cvar_define", "data_type": "i32", "variable": "var_i32"},
+            {"data": "cvar_define", "data_type": "u32", "variable": "var_u32", "size": 32},
+            {"data": "cvar_define", "data_type": "i64", "variable": "var_i64"},
+            {"data": "cvar_define", "data_type": "u64", "variable": "var_u64", "size": 64},
+            {"data": "cvar_define", "data_type": "i32", "variable": "var_i32neg"},
+            {"data": "cvar_define", "data_type": "i64", "variable": "var_i64neg"},
```

Comment:
I omit `"size"` in the signed variables above, which currently fails PHIR validation. Created a PR to allow this (the spec already marks the size field as optional): https://github.com/CQCL/phir/pull/84